packages/webapp/pages/onboarding.tsx

```diff
@@ -358,7 +358,12 @@ export function OnboardPage(): ReactElement {
   }
 
   return (
-    <div className="z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden">
+    <div
+      className={classNames(
+        'z-3 flex h-full max-h-screen min-h-screen w-full flex-1 flex-col items-center overflow-x-hidden',
```

Comment:
Technically you want to swap the max-h no?

Something like:
```
isCTA ? 'max-h-dvh' : 'max-h-screen'
```

Actually think maybe you can just use max-h-dvh everywhere no?

Reply:
> Actually think maybe you can just use max-h-dvh everywhere no?
@rebelchris You mean just replace the max-h-screen / min-h-screen with dvh equivalents entirely?

Reply:
Yeah would that work?
Otherwise above ternary maybe neater?

Reply:
@rebelchris That did seem to work, indeed. 

I did the onboarding flow a couple of times to check if all is in order. It seemed to look as it should to me.